contributed_definitions/nyaml/NXapplied_pressure.yaml

```diff
@@ -0,0 +1,22 @@
+category: base
+doc: |
+  Pressure applied to sample.
+symbols:
+  n_shear: |
+    number of values in applied pressure
+type: group
+NXapplied_pressure(NXexternal_stimulus):
+  pressure(NX_FLOAT):
+    unit: NX_PRESSURE
+    doc: |
+      Applied pressure. Could a a single float value for static field or  
```

Comment:
I would rather think about a (nx, ny, nz, 3, 3) field for e.g. a field quantity with a tensor at each point, these are typical pieces of information collected e.g. in the AddMorePower project